src/utils/parse-vitest-fn-call.ts

```diff
@@ -128,9 +127,7 @@ const determineVitestFnType = (name: string): VitestFnType => {
 
   if (name === 'expectTypeOf') return 'expectTypeOf'
 
-  if (name === 'vi') return 'vi'
-
-  if (name === 'vitest') return 'vitest'
+  if (name === 'vi' || name === 'vitest') return 'vi'
```

Comment:
is there any reason for why you're returning `vi` here instead of `vitest`?  I think we should return vitest since it's the orginal type.

Reply:
fair!